packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -53,6 +64,10 @@ export interface ParticipantItemProps {
    * ```
    */
   styles?: ParticipantItemStylesProps;
+  /**
+   * Optional strings to override in component
+   */
+  strings?: ParticipantItemStrings;
```

Comment:
Should this also have `Partial<ParticipantItemStrings>` to match the other interfaces?